test/FunctionalTests/Tests/DataOData/Tests/OData.Writer.Tests/Writer/WriterFeedTests.cs

```diff
@@ -249,7 +249,7 @@ public void FeedInlineCountTests()
                 });
         }
 
-        [TestMethod, Variation(Description = "Test feeds with invalid content.")]
+//        [TestMethod, Variation(Description = "Test feeds with invalid content.")]

```

Comment:
> [TestMethod, [](start = 9, length = 13)

Do we need to fix it?

Reply:
So, it's ready to revert?

---
In reply to: [259090571](https://github.com/OData/odata.net/pull/1204#discussion_r259090571) [](ancestors = 259090571)

Reply:
Opened [https://github.com/OData/odata.net/issues/1427](https://github.com/OData/odata.net/issues/1427) to track re-enabling disabled tests.

---
In reply to: [266672982](https://github.com/OData/odata.net/pull/1204#discussion_r266672982) [](ancestors = 266672982,259090571)